src/index.js

```diff
@@ -56,6 +56,15 @@ const MicroModal = (() => {
     }
 
     showModal () {
+      const isOpenClassName = 'is-open'
```

Comment:
Lines 59-63 check to see if the modal is already open.  If so, then it returns from the `showModal` method